programs/programs/policyengine/tests/test_policy_engine.py

```
@@ -0,0 +1,2548 @@
+
+        # Head of household with no income
+        no_income_head = HouseholdMember.objects.create(
+            screen=no_income_screen,
+            relationship="headOfHousehold",
+            age=30,
+            disabled=False,
+            student=False,
+        )
+
+        # Child with no income
+        no_income_child = HouseholdMember.objects.create(
```

Ruff (0.14.3)

Local variable `no_income_child` is assigned to but never used

Remove assignment to unused variable `no_income_child`

(F841)